add 115-async-api (#552)
openvinotoolkit/openvino_notebooks: notebooks/115-async-api/README.md

# Runtime Inference Optimization with Async API

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F115-async-api%2F115-async-api.ipynb)

This notebook demostrate how to use [asynchronous execution](https://docs.openvino.ai/nightly/openvino_docs_deployment_optimization_guide_common.html) to improve data pipelining. 

> OpenVINO™ Runtime supports inference in either synchronous or asynchronous mode. Using the Async API can improve application’s overall frame-rate, because rather than wait for inference to complete, the app can keep working on the host, while the accelerator is busy. 

Reviewer: "the app can keep working on the host, while the accelerator is busy. " Could you elaborate a bit more on what does the host mean and what the accelerator stands for here?

Reviewer: Maybe could add the explanation of how the sync mode and the async mode works (such as the figures included in the Jupyter notebook) in the Readme as well, to help the readers to learn what the difference is when entering this notebook folder.

Author: Thanks for your time to review, the content is updated accordingly

> # Runtime Inference Optimization with Async API

Reviewer 2: ```suggestion
# Asynchronous Inference with OpenVINO™
```

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F115-async-api%2F115-async-api.ipynb)

> This notebook demostrate how to use [asynchronous execution](https://docs.openvino.ai/latest/openvino_docs_deployment_optimization_guide_common.html) to improve data pipelining. 

Reviewer 2: ```suggestion
This notebook demonstrates how to use the [Async API](https://docs.openvino.ai/nightly/openvino_docs_deployment_optimization_guide_common.html) for asynchronous execution with OpenVINO. 
```

> OpenVINO™ Runtime supports inference in either synchronous or asynchronous mode. Using the Async API can improve application’s overall frame-rate, because thie function can keep the hardware busy by sending inference requests continuously. In other words, if you can afford sending a new request without waiting for the previous one to finish. 

Reviewer 2: ```suggestion
OpenVINO Runtime supports inference in either synchronous or asynchronous mode. The key advantage of the Async API is that when a device is busy with inference, the application can perform other tasks in parallel (e.g. populating inputs or scheduling other requests) rather than wait for the current inference to complete first. 
```
